packages/twenty-front/src/pages/settings/SettingsWorkspaceMembers.tsx

```diff
@@ -375,14 +375,20 @@ export const SettingsWorkspaceMembers = () => {
                     <TableCell align="right">
                       {currentWorkspaceMember?.id !== workspaceMember.id && (
                         <StyledButtonContainer>
-                          <IconButton
-                            onClick={() => {
+                          <ManageMembersDropdownMenu
+                            dropdownId={`workspace-member-actions-${workspaceMember.id}`}
+                            workspaceMemberId={workspaceMember.id}
+                            userWorkspaceId={
+                              (
+                                workspaceMember as unknown as {
+                                  userWorkspaceId?: string;
+                                }
+                              ).userWorkspaceId
+                            }
+                            onRequestDelete={() => {
```

Comment:
we do not want to drill down callbacks for components that are business related. Let's implement the behavior directly in the component.

onAction callbacks are mostly used for UI components that non business related. Otherwise it's complexify the devXP